### PR TITLE
New landing page

### PR DESCRIPTION
**Pull request by laylapku**
Updated landing page based on latest design on Zeplin.

### CODE REVIEW

File: src/components/banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { color } from "../../shared/style";
+
+const BannerWrapper = styled.div`
+  height: 40px;
+  width: max-content;
```
Comment (egor-sorokin) [inline src/components/banner/Banner.jsx:8]: as far as I can see it won't work in IE11 https://caniuse.com/#search=max-content and IE 11 is still in use, unfortunately 

- [ ]  replace width: max-content; for the app support in IE11

File: src/components/banner/Banner.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { text } from "@storybook/addon-knobs";
+import StoryWrapper from "../story-wrapper/StoryWrapper";
+import Banner from "./Banner";
+
+export default {
+  title: "Design System|Banner",
+  component: Banner,
+};
+
+export const primary = () => {
```
Comment (egor-sorokin) [inline src/components/banner/Banner.stories.jsx:11]: Looks like a component name, why does in start from lower case?

File: src/components/details/Details.jsx [hunk 3]
```diff
@@ -75,9 +65,9 @@ const Details = ({ children, title }) => {
     <DetailsContainer onToggle={handleOpen}>
       <StyledSummary>
         {title}
-        <IconWrapper>
+        <div>
```
Comment (egor-sorokin) [inline src/components/details/Details.jsx:68]: I am just curious do you really need this div? Will the icon be displayed correctly without it? 

- [ ] check div element

File: src/components/tool-overview/ToolOverview.jsx [hunk 4]
```diff
@@ -37,38 +43,64 @@ const ToolLink = styled(Link)`
   align-items: center;
 `;
 
-const ToolOverview = ({ toolData }) => {
-  const { id, title, img, desc, details, ctaData, linkData } = toolData;
+const ScriptWrapper = styled(Stack)`
+  & > * + * {
```
Comment (egor-sorokin) [inline src/components/tool-overview/ToolOverview.jsx:47]: try to use more specific selectors, than '*', do you really need to set a margin on every element inside of ScriptWrapper?

- [ ] try to use more specific selectors

File: src/components/banner/Banner.jsx (hunk 1 above)
Comment (laylapku) [reply src/components/banner/Banner.jsx:8]: I updated with the value suggested in Zeplin design file. Thanks for the tip!

File: src/components/banner/Banner.stories.jsx (hunk 2 above)
Comment (laylapku) [reply src/components/banner/Banner.stories.jsx:11]: I think it's the convention of how the component stories are written from the start so I'm following it for consistency purpose. In the storybook GUI though these names will be displayed as capitalized.

File: src/components/details/Details.jsx (hunk 3 above)
Comment (laylapku) [reply src/components/details/Details.jsx:68]: The reason is that as the screen size gets small the svgs will likely get squeezed by the text to its left. So sizes of these svgs on different lines would be different, depending on the lengths of their sibling text elements. Adding a wrapping `div` would contain the svg in a block so the size would remain consistent. Any better solutions for this?

Comment (egor-sorokin) [reply src/components/details/Details.jsx:68]: hm, since it protects from some weird behaviour let's keep it

**Pull request merged**